sql/core/src/test/scala/org/apache/spark/sql/execution/SparkSqlParserSuite.scala

```diff
@@ -259,4 +259,22 @@ class SparkSqlParserSuite extends AnalysisTest {
       parser.parsePlan("ALTER SCHEMA foo SET DBPROPERTIES ('x' = 'y')"))
     assertEqual("DESC DATABASE foo", parser.parsePlan("DESC SCHEMA foo"))
   }
+
+  test("manage resources") {
+    assertEqual("ADD FILE abc.txt", AddFileCommand("abc.txt"))
+    assertEqual("ADD FILE \'abc.txt\'", AddFileCommand("abc.txt"))
```

Comment:
is `\'` the same as `'` here?

Reply:
yes.

Reply:
shall we use `'`? it's easier to read

Reply:
@cloud-fan Yes, Its better that way. I have made the changes.